pages/deployment/gcp.mdx

```diff
@@ -0,0 +1,177 @@
+in your Memgraph database, and the quantity of the snapshots you want to keep
+alive. 
+
+The general rule of thumb is to double the storage than the amount of data you
+will store. If you are going to store 100GB of data, you should have at least
+200GB of disk storage. 
+
+
+## Installing Memgraph 
+
+Depending on the way you want to deploy Memgraph, native or via Docker, you need
+to follow the steps below: 
+
+<Tabs items={["Docker" , "Linux"]}>
+
+ <Tab> 
+ 
+    For example, if you are using Ubuntu 20.04, you can follow the steps
+    below to install Docker:
+
+        ```bash
+            sudo apt-get update
+            sudo apt-get install apt-transport-https ca-certificates curl gnupg-agent software-properties-common
+            sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu focal stable"
+            sudo apt-get update
+            sudo apt-get install docker-ce docker-ce-cli containerd.io
+            docker --version
+        
+        ```
+    Note that the steps above may vary depending on your exact Linux distribution.
+
+    After the docker is installed, you can pull the Memgraph image and run it:
+
+        ```bash
+            docker run -p 7687:7687 memgraph/memgraph:latest
+        ```
+
+    This will run Memgraph on the default port 7687. You should be able to connect
+        to it via [Memgraph Lab](../data-visualization) or via [client
+    libraries](../client-libraries) using the bolt protocol. If you have an issue
+    connecting to Memgraph remotely, make sure that the port 7687 is open in [the
+    firewall rules on your GCP VM](#Setup-the-network). 
+ </Tab>
+ 
+ <Tab>
+
+ Depending on the Linux distribution you are using, you can follow the guide for
+ installing Memgraph on each distribution: 
+
+    - [Ubuntu](../getting-started/install-memgraph/ubuntu.mdx)
+    - [Fedora](../getting-started/install-memgraph/fedora.mdx)
+    - [Debian](../getting-started/install-memgraph/debian.mdx)
+    - [Centos](../getting-started/install-memgraph/centos.mdx)
+    - [Amazon Linux](../getting-started/install-memgraph/amazon-linux.mdx)
+    - [RedHat](../getting-started/install-memgraph/redhat.mdx)
+    - [Rocky](../getting-started/install-memgraph/rocky.mdx)
+
+ </Tab> 
+ 
+ </Tabs>
+
+## Manage Memgraph deployment
+
+After Memgraph is installed and running on your GCP VM, Memgraph management in
+GCP VM is identical to the general guidelines that are described in the form of
+a native Linux Memgraph or Docker container Memgraph. 
+
+Depending on what you are using, you can follow the [Linux](./linux.mdx) or
+[Docker](./docker.mdx) deployment guide for more information on how to manage
+the Memgraph deployment.
+
+
+# Next steps
+
+Memgraph also supports deployment in the Kubernetes environment. If you are
+interested in deploying Memgraph in Kubernetes, you can follow the [Memgraph
+Kubernetes installation
+guide](../getting-started/install-memgraph/kubernetes.mdx). 
+
+
```

Comment:
Maybe add OH section here, as in other guides, e.g. https://documentation-bqpg3d7vp-memgraph.vercel.app/docs/deployment/docker#where-to-next. This comment applies to other guides as well.

Reply:
Done